Navaids/VOR_Non-UK.txt

```diff
@@ -10,41 +10,41 @@ RTM 110.400 N051.58.25.310 E004.28.51.490 ; Rotterdam
 SPL 108.400 N052.19.55.720 E004.44.59.580 ; Schiphol
 SPY 113.300 N052.32.25.010 E004.51.13.610 ; Spijkerboor (DME)
 
-;Belgium, verified 2012/01
+;Belgium, verified 2308
 AFI 114.900 N050.54.28.000 E004.08.20.000 ; Affligem
 ANT 113.500 N051.11.26.000 E004.28.21.000 ; Antwerpen
-BBE 116.000 N050.45.23.000 E004.46.05.000 ; Beauvechain (TACAN Ch107X)
-BBL 108.600 N051.10.13.000 E005.27.59.000 ; Kleine-Brogel (TACAN Ch33X)
-BFS 110.500 N050.14.29.000 E004.39.11.000 ; Florennes (TACAN Ch52X)
+BBE 116.000 N050.45.25.000 E004.46.07.000 ; Beauvechain (TACAN Ch107X)
+BBL 108.600 N051.10.03.000 E005.27.51.000 ; Kleine-Brogel (TACAN Ch33X)
+BFS 110.500 N050.14.29.000 E004.39.12.000 ; Florennes (TACAN Ch52X)
 BUB 114.600 N050.54.08.000 E004.32.17.000 ; Brussels
 BUN 110.600 N051.07.07.000 E004.50.32.000 ; Bruno
-CIV 113.200 N050.34.26.000 E003.49.58.000 ; Chi�vres
+CIV 113.200 N050.34.26.000 E003.49.58.000 ; Chièvres
 COA 110.050 N051.20.53.000 E003.21.19.000 ; Costa
 DIK 114.400 N049.51.41.000 E006.07.47.000 ; Diekirch
 FLO 112.050 N050.52.36.000 E005.08.04.000 ; Flora
 GSY 115.700 N050.27.14.000 E004.26.29.000 ; Gosly
 HUL 117.550 N050.44.58.000 E004.38.30.000 ; Huldenberg
 KOK 114.500 N051.05.41.000 E002.39.06.000 ; Koksy
-LGE 115.450 N050.39.14.000 E005.28.14.000 ; Li�ge
+LGE 115.450 N050.39.14.000 E005.28.14.000 ; Liège
 LNO 112.800 N050.35.09.000 E005.42.37.000 ; Olno
 LUX 112.250 N049.38.22.000 E006.14.50.000 ; Luxembourg
 NIK 117.400 N051.09.54.000 E004.11.02.000 ; Nicky
 SPI 113.100 N050.30.53.000 E005.37.25.000 ; Sprimont
 
-;France
+;France, verified AIRAC 2308
 ABB 108.450 N050.08.06.500 E001.51.16.900 ; Abbeville
-AMB 113.700 N047.25.44.100 E001.03.52.000 ; Amboise
+AMB 113.700 N047.25.44.100 E001.03.52.000 ; Amboise (DME)
 BNE 113.800 N050.37.29.200 E001.54.25.600 ; Boulogne Sur Mer
-CAN 115.400 N049.10.23.000 W000.27.19.000 ; Caen Carpiquet
+CAN 115.400 N049.10.23.500 W000.27.19.000 ; Caen Carpiquet
 CMB 112.600 N050.13.41.300 E003.09.05.400 ; Cambrai Epinoy
 CLM 112.900 N048.50.40.700 E003.00.49.900 ; Coulommiers Voisins
 DVL 110.200 N049.18.38.700 E000.18.45.800 ; Deauville Saint Gatien
 DPE 115.800 N049.55.31.400 E001.10.14.300 ; Dieppe
-DIN 114.300 N048.35.10.000 W002.04.56.000 ; Dinard Pleurtuit Saint Malo
+DIN 114.300 N048.35.09.800 W002.04.56.400 ; Dinard Pleurtuit Saint Malo
 EVX 112.400 N049.01.54.100 E001.13.15.100 ; Evreux Fauville
 LGL 115.000 N048.47.26.200 E000.31.49.000 ; L'Aigle
 LDV 115.150 N048.31.41.800 W004.09.22.700 ; Landivisiau (TACAN)
-BST 114.700 N048.17.04.600 W004.27.07.600 ; Lanveoc Poulmic
+BST 114.700 N048.17.04.300 W004.27.07.600 ; Lanveoc Poulmic (TACAN)
```

Comment:
```suggestion
BST 114.700 N048.17.04.300 W004.27.07.600 ; Lanveoc Poulmic (TACAN Ch93X)
```
Not sure why we bother, but for consistency with others above.

Reply:
Yeah I think we should just remove the channel, we don't do it anywhere else.

Reply:
interesting, I just saw it on the Belgian ones, no biggie but probably doesn't add much value so I'm not bothered either way personally.

Reply:
I'll take them out 👍